pages/api/graphql.js

```diff
@@ -1,7 +1,15 @@
 import { ApolloServer } from 'apollo-server-micro';
-import { schema } from '@/utils/apollo/schema';
+import { typeDefs } from '@/utils/apollo/schema/typeDefs';
+import { resolvers } from '@/utils/apollo/schema/resolvers';
+import dbConnect from '@/utils/dbConnect';
 
-const apolloServer = new ApolloServer({ schema });
+const apolloServer = new ApolloServer({
+  typeDefs,
+  resolvers,
+  context: async ({ req }) => {
+    await dbConnect();
+  },
```

Comment:
oh. Does this work? I didn't know that. Can you send me the source where you got this from?

Reply:
https://www.apollographql.com/docs/apollo-server/data/resolvers/#the-context-argument
Since we're not storing our database connection in a variable, we just need to connect to the database

Reply:
lovely <3